modules/openlmis-web/src/main/webapp/public/js/distribution/model/adult-coverage.js

```diff
@@ -20,6 +20,33 @@ function AdultCoverage(facilityVisitId, adultCoverageJSON) {
   $(this.openedVialLineItems).each(function (i, lineItem) {
     _this.openedVialLineItems[i] = new OpenedVialLineItem(lineItem);
   });
+
+  var fieldList = ['healthCenterTetanus1', 'outreachTetanus1', 'healthCenterTetanus2To5', 'outreachTetanus2To5'];
+
+  function countNRStatus(lineItems, fields) {
```

Comment:
I really like the fact that you put this logic into a function. I notice that the same function is also defined in child-coverage.js, though. Would it be possible to define the function just once, in a shared location, and to reuse it?